superset/assets/src/explore/components/ExploreActionButtons.jsx

```diff
@@ -24,23 +24,25 @@ import { t } from '@superset-ui/translation';
 import URLShortLinkButton from '../../components/URLShortLinkButton';
 import EmbedCodeButton from './EmbedCodeButton';
 import DisplayQueryButton from './DisplayQueryButton';
-import { exportChart, getExploreLongUrl } from '../exploreUtils';
+import { exportChart, getExploreLongUrl, exportChartPNG} from '../exploreUtils';
 
 const propTypes = {
   actions: PropTypes.object.isRequired,
   canDownload: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]).isRequired,
   chartStatus: PropTypes.string,
   latestQueryFormData: PropTypes.object,
   queryResponse: PropTypes.object,
+  slice: propTypes.slice
```

Comment:
This is referring to itself.
Please check `prop-types` reference on how to define a proper prop type.